package/cobalt/cobalt.mk

```diff
@@ -1,40 +1,62 @@
 COBALT_VERSION = 06cbe09f396bfbcac0330c336180150352af2a94
 COBALT_SITE = https://github.com/Metrological/cobalt
 COBALT_SITE_METHOD = git
+COBALT_INSTALL_STAGING = YES
+COBALT_DEPENDENCIES = host-bison alsa-lib
+
 export BUILDROOT_HOME=$(HOST_DIR)/usr
 export PATH := $(HOST_DIR)/bin:$(HOST_DIR)/usr/bin:$(HOST_DIR)/usr/sbin:$(PATH)
 
-COBALT_DEPENDENCIES = alsa-lib
-ifeq ($(BR2_PACKAGE_COBALT_WITH_FFMPEG),y)
+ifeq ($(BR2_PACKAGE_COBALT_WITH_FFMPEG), y)
```

Comment:
Is there a reason to keep FFMPEG support? Else I'd rather clean it up and make it Gstreamer only. Keeping the recipes/makes files simple.

Reply:
Sorry I am not sure about this. This support is already in the cobaltt.mk, I think @srinivas-kakarla  is the best person to give answer on this. @srinivas-kakarla, can you please give your input on this ?

Reply:
@wouterlucas,
We can remove FFMPEG support from the Cobalt browser and make it only GStreamer based.

Reply:
Thank you @srinivas-kakarla, then we will remove ffmpeg support from the .mk